stage1/init/common/pod.go

```diff
@@ -59,6 +60,37 @@ var (
 		"LOGNAME": "root",
 		"HOME":    "/root",
 	}
+
+	// List of default capabilities inside systemd-nspawn pod is available
+	// here: https://www.freedesktop.org/software/systemd/man/systemd-nspawn.html
+	stdCapabilities = []types.LinuxCapability{
+		"CAP_CHOWN",
+		"CAP_DAC_OVERRIDE",
+		"CAP_DAC_READ_SEARCH",
+		"CAP_FOWNER",
+		"CAP_FSETID",
+		"CAP_IPC_OWNER",
+		"CAP_KILL",
+		"CAP_LEASE",
+		"CAP_LINUX_IMMUTABLE",
+		"CAP_NET_BIND_SERVICE",
+		"CAP_NET_BROADCAST",
+		"CAP_NET_RAW",
+		"CAP_SETGID",
+		"CAP_SETFCAP",
+		"CAP_SETPCAP",
+		"CAP_SETUID",
+		"CAP_SYS_ADMIN",
+		"CAP_SYS_CHROOT",
+		"CAP_SYS_NICE",
+		"CAP_SYS_PTRACE",
+		"CAP_SYS_TTY_CONFIG",
+		"CAP_SYS_RESOURCE",
+		"CAP_SYS_BOOT",
+		"CAP_AUDIT_WRITE",
+		"CAP_AUDIT_CONTROL",
+		"CAP_MKNOD",
```

Comment:
Do we need to copy the full list here? It should already be defined in gocapability:
https://github.com/coreos/rkt/blob/master/Godeps/_workspace/src/github.com/syndtr/gocapability/capability/enum.go#L39